deploy/requirements.md

```diff
@@ -223,6 +223,11 @@ Please also note that the worker nodes will need the right permission to describ
     "Resource": "*",
     "Effect": "Allow"
 },
+{
+    "Action": "cloudformation:Update*",
```

Comment:
Is this a new requirement or forgot?
If it's new please add also a section for updating to the new version in the README.md

Reply:
It was missing for stack updates but also covers `cloudformation:UpdateTerminationProtection`. I think it's good enough that the requirements.md are updated so it will work whether you use this feature or not.